File: backend-model/src/main/java/outcobra/server/model/dto/TimetableDto.java
package outcobra.server.model.dto;

import outcobra.server.model.Timetable;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class TimetableDto implements MappableDto<TimetableDto, Timetable> {
    @Override
    public Mapper<TimetableDto, Timetable> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public Timetable toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/TimetableDto.kt
package outcobra.server.model.dto

import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.Timetable
import outcobra.server.model.mapper.Mapper

/**
 * Created by Florian on 04.11.2016.
 */
class TimetableDto : MappableDto<TimetableDto, Timetable> {
    override fun getMapper(): Mapper<TimetableDto, Timetable> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): Timetable {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}